sphinx/application.py

```diff
@@ -29,7 +28,7 @@
     directives, roles
 
 import sphinx
-from sphinx import package_dir, locale
+from sphinx import package_dir, locale, extensions
```

Comment:
Either is fine. But sphinx already uses `sphinx.domains`, `sphinx.directives`, `sphinx.writers` and so on.
only `locale` and `util` are singular.

If we change it to `sphinx.extension`, we should rename to `sphinx.extensions:confirm()` to better one. any idea?